bin/setup.sh

```diff
@@ -55,8 +55,8 @@ echo
 echo "Database configuration"
 echo
 
-[[ -z $DBNAME ]] && read -p "New OPUS Database Name [opusdbfw]: "           DBNAME
-[[ -z $DB_ADMIN ]] && read -p "New OPUS Database Admin Name [opus4adminfw]: " DB_ADMIN
+[[ -z $DBNAME ]] && read -p "New OPUS Database Name [opusdb]: "           DBNAME
```

Comment:
It would be very useful if all the variables like `$DBNAME` could be set as parameters of the script. How complicated would that be? It would be good, if we could use a common parameter like `-D` for java, where you can use `-D[parameter_name]=value`.